src/app/players/epub/epub.component.ts

```diff
@@ -22,22 +21,18 @@ export class EpubComponent implements OnInit {
   playerConfig: any;
   isLoading = true;
   context =  this.configService.playerConfig.PLAYER_CONTEXT;
-  config = {
-    traceId: '123456',
-    sideMenu: {
-      showShare: true,
-      showDownload: true,
-      showReplay: false,
-      showExit: false
-    }
-  };
+  epubMetaDataconfig: any = JSON.parse(localStorage.getItem('epubConfig')) || {};
```

Comment:
This will break initially since data will not be there in local storage and use session storage for this, best practice to keep this in service and get it from there